Updated testcase documentation (#1069)
nunit/docs: docs/articles/nunit/writing-tests/attributes/testcase.md

  but may serve a purpose for the test author)
* **TypeArgs** specifies the `Type`s to be used when targeting a generic test method. (_NUnit 4.1+_)

## Be aware of mixing the syntax for named parameters and attributes with the same name

### Ignore done right and wrong

[!WARNING]

> When using the **Ignore** parameter (and others, see below), note that this has to be a named parameter. It is easy to add another Ignore attribute after the TestCase attribute. That will be the same as adding it separately, and it will apply to the complete fixture. Note, that this might apply to more of these named parameters, with names equal to other attributes, like the Explicit and Category parameters.

Reviewer: ```suggestion
When using the `Ignore` parameter (and others as noted below), keep in mind that this has to be a named parameter. It is easy to add another `Ignore` attribute after the `TestCase` attribute. That will be the same as adding it separately, and it would apply to the complete fixture. This also may apply to more of these named parameters, with names equal to other attributes, like the Explicit and Category parameters.
```
